pkgs/by-name/dn/dnsjit/package.nix

```
@@ -0,0 +1,39 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, pkg-config
+, makeWrapper
+, autoreconfHook
+, autoconf
+, automake
+, libpcap
+, luajit
+, lmdb
+, libck
+, gnutls
+, lz4
+}:
+
+stdenv.mkDerivation rec {
+  pname = "dnsjit";
+  version = "1.3.0";
+
+  src = fetchFromGitHub {
+    owner = "DNS-OARC";
+    repo = "dnsjit";
+    rev = "03719c6538a019a9b3c857f0ff9d031208160027";
+    sha256 = "sha256-NaJqrPWeTeCSnPGJUXn7vIGRRWwztxfbEI6Wb+q4/hc=";
+  };
+
+  nativeBuildInputs = [ pkg-config autoreconfHook libpcap luajit lmdb libck gnutls lz4 ];
```

```suggestion
  nativeBuildInputs = [
  	pkg-config
  	autoreconfHook
  	libpcap
  	luajit
  	lmdb
  	libck
  	gnutls
  	lz4
  ];
```
please do some formatting so it's easier to read, and are you sure all of those belong into nativeBuildinputs?